fillin/src/main/java/com/teamfillin/fillin/domain/account/Account.java

```diff
@@ -0,0 +1,34 @@
+package com.teamfillin.fillin.domain.account;
+
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Account {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long no;
+
+	@Embedded
+	private SocialInfo socialInfo;
+
+	@Column(length = 500)
+	private String refreshToken;
+
+	@Column(nullable = false, unique = true)
```

Comment:
```suggestion
	@Column(nullable = false)
```

- userNo 에 단독 unique 가 걸려있으면, N:M 구성이 어렵습니다
  - user 1개에 N 개의 SocialAccount 매핑이 될 수 없음